metricbeat/module/http/server/_meta/docs.asciidoc

```diff
@@ -2,12 +2,15 @@ This is the server metricset of the module http.
 
 Events sent to the http endpoint will be put by default under the `http.server` prefix. To change this use the `server.paths`
 config options. In the example below every request to `/foo` will be put under `http.foo`.
-
+You can now configure ssl on top of it.
```

Comment:
This last one and we are done :-)

Instead of `You can now configure SSL on top of it`, lets write something that will last for future releases
`Also consider using secure settings for the server using TLS/SSL as shown`

Reply:
Much better